examples/datadog-express-apm/host-agent/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "datadog-express-apm",
+  "name": "host-agent",
```

Comment:
I believe it's better if you add something related to Datadog like "datadog-host-apm"

Reply:
It is already in a directory called `examples/{datadog-express-apm}/host-agent/package.json` so it's irrelevant here.